src/toil_rnaseq_sc/test/test_rnaseq_sc_cgl.py

```diff
@@ -1,46 +1,197 @@
+            "plots directory should exist in output tarball")
+    require(len(os.listdir(os.path.join(output_dir, uuid, "plots"))) > 0,
+            "plots directory should not be empty in output tarball")
+
+
+def test_pipeline_output_without_graphs(tmpdir):
+    uuid = "test_rnaseqsc_ng"
+    output_dir = os.path.join(str(tmpdir), uuid)
+    if not os.path.isdir(output_dir):
+        os.makedirs(output_dir)
+    output_file = os.path.join(output_dir, uuid + ".tar.gz")
+
+    jobstore = os.path.join(str(tmpdir), uuid + "_jobstore")
+
+    input = _get_test_fastq_files(tmpdir, tarball=False)
+    config = _generate_config(tmpdir, output_dir, generate_graphs=False)
+    manifest = _generate_manifest(tmpdir, [[uuid, input]])
+
+    subprocess.check_call(['toil-rnaseq-sc', 'run',
+                                '--config', config,
+                                '--manifest', manifest,
+                                jobstore])
+    # ensure file and directories exist (or don't)
+    require(os.path.isfile(output_file), "expected outputfile to exist: " + output_file)
+    subprocess.check_call(['tar', '-xvf', output_file, '-C', output_dir])
+    require(os.path.isfile(os.path.join(output_dir, uuid, "matrix.tsv")),
+            "matrix.tsv file should exist in output tarball")
+    require(not os.path.isdir(os.path.join(output_dir, uuid, "plots")),
+            "plots directory should not exist in output tarball")
 
-    TOIL_SCRIPTS_TEST_TOIL_OPTIONS - a space-separated list of additional command line arguments to pass to Toil via
-    the script entry point. Default is the empty string.
 
-    TOIL_SCRIPTS_TEST_JOBSTORE - the job store locator to use for the tests. The default is a file: locator pointing
-    at a local temporary directory.
+def _generate_manifest(tmpdir, list_of_lines):
+    manifest_location = os.path.join(str(tmpdir), "manifest-toil-rnaseqsc-test.tsv")
+    # clear old manifest if it exists
+    if os.path.isfile(manifest_location):
+        os.remove(manifest_location)
+    with open(manifest_location, 'w') as manifest:
+        for line in list_of_lines:
+            manifest.write("\t".join(line) + "\n")
+    return manifest_location
 
-    TOIL_SCRIPTS_TEST_NUM_SAMPLES - the number of sample lines to generate in the input manifest
-    """
+def _generate_config(tmpdir, output_dir, generate_graphs):
+    tmpdir_str = str(tmpdir)
+    config_location = os.path.join(tmpdir_str, 'config-toil-rnaseqsc-test.yaml')
+    if os.path.isfile(config_location):
+        os.remove(config_location)
+    with open(config_location, 'w') as f:
+        f.write(textwrap.dedent("""
+        kallisto-index: file:///home/trevorp/genomic/cgl/testing_data/in/transcripts.idx
+        output-dir: {output_dir}
+        generate-graphs: {generate_graphs}
+        barcode-length: 14
+        window-min: 500
+        window-max: 5000
+        sample-idx: [ATCGCTCC]
+        ci-test: true
```

Comment:
I think we can remove the `ci-test` attribute from the config (and subsequently this test file).